src/ConnectedMode/Resources.resx

```diff
@@ -147,6 +147,22 @@
   <data name="BranchProvider_NotInConnectedMode" xml:space="preserve">
     <value>[ConnectedMode/BranchMapping] Solution is not bound</value>
   </data>
+  <data name="GitMonitor_EventError" xml:space="preserve">
+    <value>[ConnectedMode] Error handling repo change notification: {0}</value>
```

Comment:
This message just has `[ConnectedMode]` as a prefix i.e. it's not `[ConnectedMode/xxxx]`.

That's because it's a "normal" verbosity message. So far, I've only used the `[ConnectedMode/xxxx]` form for "verbose" messages.